README.md

```diff
@@ -3,3 +3,91 @@
 HyperUI is a collection of free Tailwind CSS components that can be used in your next project. With a range of components, you can build your next marketing website, admin dashboard, eCommerce store and much more.
 
 ![](https://hyperui.dev/og.jpg)
+
+## Table of Contents
+- [Getting Started](#getting-started)
+- [Usage](#usage)
+- [Components](#components)
+    - [🤖 Application UI](#application-ui)
+    - [📣 Marketing](#marketing)
+    - [🛍️ eCommerce](#ecommerce)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Getting Started
+Hyper UI doesn't require any installation. All you need to do is browse components you need to implement in your project, copy its code, and simply paste it to your project.
+
+## Usage
+HyperUI offers a wide range of components suitable for various use cases, including application UI, marketing, and ecommerce. Here's an example of how to use a HyperUI component in your project:
+
+1. Choose the component you wish to use.
+2. Preview how it will appear on different screen sizes by clicking on the available screen size options. You can also check its compatibility with LTR (left-to-right) layouts and dark mode.
+
+    ![hyperui1](https://github.com/ToobaJamal/hyperui/assets/52610124/085db001-5d8d-43ca-bcef-fe7a8cc35b0a)
+
+3. Click on the 'Copy' button to copy the source code.
+
+https://github.com/ToobaJamal/hyperui/assets/52610124/0ec03263-ebda-4e84-804c-126a1a2e5056
+
+5. Paste the copied code into your project.
```

Comment:
Can this be simplified? I don't think we need the images or videos, maybe something like:

1. Browse the website for a component you like
2. Preview the component at different breakpoints and check its compatibility with:
- LTR
- Dark mode
- Interactivity with Alpine JS
3. Use the "Copy" button to copy the code to clipboard
4. Past the copied code into your project

What do you think?